reference/tbp-api.yml

```diff
@@ -545,4 +541,32 @@ components:
       required:
         - status
         - title
+    User:
+      title: User
+      type: object
+      x-examples: {}
+      properties:
+        type:
+          type: string
+          enum:
+            - user
+        attributes:
+          type: object
+          properties:
+            state:
```

Comment:
I wonder if `state` should be required for this?  If it's not required, how would our endpoint respond if it was missing? with success (as in a no-op, but the request was successful) or error?
@nrocy thoughts?

Reply:
good question. Also if we think that we would have more than just state attribute property with collection of errors we would need to point to specific property in the response. That could be problematic couldnt be

Reply:
hmm i'm kinda unsure, given it's potentially one of a few fields down the line i'd not want to make it mandatory now and then have to change it later. it could be a NOOP but for PUT/POST it'd nuke the state if it were empty, right?

so i'd say make it optional but the only defined operation being PATCH for now as this feels like the intent we have - to do partial updates to user attribute values